lib/cog/chat/hipchat/template_processor.ex

```diff
@@ -33,6 +33,8 @@ defmodule Cog.Chat.HipChat.TemplateProcessor do
     do: "<pre>#{text}</pre>"
   defp process_directive(%{"name" => "paragraph", "children" => children}),
     do: Enum.map_join(children, &process_directive/1) <> "<br/><br/>"
+  # If you try to render a link with nil or blank text, nothing is displayed to the user.
+  # This way at least a link is rendered.
```

Comment:
Clarifying that HipChat's default behavior is to render nothing with nil/blank text would be nice. At first, that wasn't quite clear, and this comment was a little confusing; for a second, I thought you were saying that it was Cog's default behavior.